apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -63,13 +63,10 @@ loop(Socket, Transport, State) ->
                     InfoBin = dproto_tcp:encode_bucket_info(Info),
                     Transport:send(Socket, InfoBin),
```

Comment:
Remove the dynamic function call on line 64. Only modules that define callbacks should make dynamic calls.